chore: configure travis ci

theallegrarr opened the pull request:
- add .travis.yml file with basic configuration
- test run and passed in travis

twumm committed:
chore: Setup react next app

Install initial packages required for the app
Create base folder structure in components folder using atomic design
Create pull request template

theallegrarr committed:
chore(install dependencies): install and configure husky and commitlint

twumm committed:
chore: remove nav file from default next.js file

twumm committed:
chore: setup SSR styling config for styled-components

Install babel-plugin-styled-components
Pass styled-components as a plugin in .babelrc
Configure style sheet to be passed as prop in _document.js in the pages folder

twumm committed:
chore: configure storybook

Create a button with a story
Configure where stories will be loaded from

twumm committed:
chore: add eslint

Install dependencies for eslint
Add airbnb as a guide for the eslint

twumm committed:
chore: write tests

Write tests for the button component
Remove title tag from the _document.js file

twumm committed:
chore: write passing test for Button component

Install jest-next-dynamic
Create test for Button component by getting text in button

theallegrarr committed:
chore: add and configure travis ci

theallegrarr committed:
chore: integrate build status badge

theallegrarr commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Configure integration testing](https://trello.com/c/To6HaVeq/33-configure-integration-testing)

theallegrarr committed:
chore: add hound yml config

theallegrarr committed:
chore: update hound config

theallegrarr committed:
chore: update eslint config

theallegrarr committed:
chore: hound check

abidex4yemi reviewed (changes requested):
Please resolve the hound ci yml issue

twumm committed:
chore: create an eslint json file

Update all config in eslint with rules and config from our eslintrc.js file
Using an eslint file with json is required for hound ci to pass as the service only uses YAML or JSON file types for the eslint

twumm committed:
refactor: refactor eslint and hound files

theallegrarr committed:
chore: remove hound.yml file

theallegrarr committed:
Merge branch 'chore-configure-travis-ci' of github.com:LABS-EU3/frontend_community_calendar into chore-configure-travis-ci
